styles.css

```diff
@@ -891,23 +920,20 @@ button:active {
 
 /* styling for pop-up window in the desktop version */
 .popUp {
-  position: absolute;
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
- [ ] Please kindly fix the pop-up live button and project button 
### **Users can't see the button**
![butt](https://user-images.githubusercontent.com/55596970/169628185-7f0c5a7e-6038-42a6-8567-3858febc7024.png)